views/includes/header.ejs

```diff
@@ -2,14 +2,18 @@
   <nav>
     <ul>
       <li><a href="/">Home</a></li>
+      <% if (!locals.isAuth) { %>
       <li><a href="/signup">Signup</a></li>
       <li><a href="/login">Login</a></li>
+      <% } %> <% if (locals.isAuth) { %>
```

Comment:
missing new line between end of first if and start of second if?
Could also use an if else